data/expenditureexpert.json

```diff
@@ -1,31 +1,39 @@
 {
-  "expenses" : [ {
-    "description" : "Took Grab",
-    "expenseCategory" : "Travel",
-    "amount" : "10"
-  }, {
-    "description" : "Took Bus",
-    "expenseCategory" : "Travel",
-    "amount" : "3"
-  }, {
-    "description" : "McDonalds",
-    "expenseCategory" : "Food",
-    "amount" : "15"
-  }, {
-    "description" : "Pizza",
-    "expenseCategory" : "Food",
-    "amount" : "30"
-  }, {
-    "description" : "Ice Cream",
-    "expenseCategory" : "Food",
-    "amount" : "4"
-  }, {
-    "description" : "Watched Movie",
-    "expenseCategory" : "Entertainment",
-    "amount" : "12"
-  }, {
-    "description" : "Took Train",
-    "expenseCategory" : "Travel",
-    "amount" : "2"
-  } ]
+  "expenses": [
+    {
+      "description": "Took Grab",
+      "expenseCategory": "Travel",
+      "amount": "10"
+    },
+    {
+      "description": "Took Bus",
+      "expenseCategory": "Travel",
+      "amount": "3"
+    },
+    {
+      "description": "McDonalds",
+      "expenseCategory": "Food",
+      "amount": "15"
+    },
+    {
+      "description": "Pizza",
+      "expenseCategory": "Food",
+      "amount": "30"
+    },
+    {
+      "description": "Ice Cream",
+      "expenseCategory": "Food",
+      "amount": "4"
+    },
+    {
+      "description": "Watched Movie",
+      "expenseCategory": "Entertainment",
+      "amount": "12"
+    },
+    {
+      "description": "Took Train",
+      "expenseCategory": "Travel",
+      "amount": "2"
+    }
+  ]
```

Comment:
is the additional of break line necessary between every brackets?

Reply:
again, i was failing ci checks